### ISSUE TITLE
GitHub Workflow for eslint

### ISSUE THREAD

**Issue by to-sta** [activist-org/activist#446]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

General:
On the backend side of things we have the whole suite already: formatter, linter and static type checking. It is also configured to work well together. 

On the frontend side we are still missing the linter. 

The task is to add an `eslint` step to the `pr_ci_frontend` workflow. In case this or any other step fails in the workflow it should continue to run.

Prerequisite is the merge of PR #444 and solving the issue #445

### Contribution

I am happy to help someone or work on this myself :smile

**Comment by to-sta** [comment]
I added another issue #449 about configuring eslint. This should be resolved before we start merging a PR with a new workflow.

**Comment by andrewtavis** [comment]
Closed by #466 🚀 Let me know if you think anything else is needed as far as workflows, @to-sta!
